perfkitbenchmarker/dpb_service.py

```diff
@@ -30,6 +30,10 @@
                     'If set, the name of the pre created dpb implementation,'
                     'assumed to be ready.')
 flags.DEFINE_string('dpb_log_level', 'INFO', 'Manipulate service log level')
+flags.DEFINE_string('dpb_job_jarfile', None,
+                    'Executable Jarfile containing workload implementation')
+flags.DEFINE_string('dpb_job_classname', None, 'Classname to be used')
```

Comment:
Is this a class in the jar file? Add context.

Reply:
Done. Provided additional documentation for the flag.